cmd/datamon/cmd/label_set.go

```diff
@@ -55,13 +43,13 @@ var SetLabelCommand = &cobra.Command{
 		if err != nil {
 			logFatalln(err)
 		}
-
 	},
 }
 
 func init() {
 	requiredFlags := []string{addRepoNameOptionFlag(SetLabelCommand)}
 
+	// todo
```

Comment:
please add what you want to finish up here.

Reply:
this was to make the flags required